app/controllers/searches_controller.rb

```diff
@@ -66,13 +65,15 @@ def advanced
 
   def pick_klass_vertical_template
     if get_commercial_results?
-      [WebSearch, :web, :index]
+      [WebSearch, :web, :index, 'searches']
     elsif gets_i14y_results?
-      [I14ySearch, :i14y, :i14y]
+      [I14ySearch, :i14y, :i14y, 'searches']
     elsif @affiliate.gets_blended_results
-      [BlendedSearch, :blended, :blended]
+      [BlendedSearch, :blended, :blended, 'searches']
+    elsif redesign?
+      [WebSearch, :web, :index_redesign, 'searches_redesign']
     else
-      [WebSearch, :web, :index]
+      [WebSearch, :web, :index, 'searches']
```

Comment:
@ddzz , I'm going to approve this for now to facilitate getting this story out the door, but I think we'll want to refactor this method. This method was already doing too many things, and now it's even more overloaded (as-is, it should be renamed to `pick_klass_vertical_template_layout`, as it's now setting 4 different things). I'm thinking that we may want a `layout` method (or `serp_layout`, if `layout` is a protected term). Eventually all the different search engines will be supported by the redesigned serp, so the only logic we should ultimately need is something like:
```ruby
format.html { render template, layout: serp_layout }
...
def serp_layout
  redesign? ? 'searches_redesign' : 'searches'
end
```

Reply:
The plan is to refactor this method as we support more search engines, so these changes are all temporary. I considered an approach similar to the `#serp_layout` method, but I decided against it for now because it would allow the redesigned layout to be used with the old templates for I14y and blended searches.